tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -114,10 +115,10 @@ def __init__(self, vm: VmIdentifier, paths: AgentTestContext.Paths, connection:
             super().__init__(vm=vm, paths=paths, connection=connection)
             # These are initialized by AgentTestSuite._set_context().
             self.log_path: Path = None
-            self.log: Logger = None
+            self.lisa_log: Logger = None
             self.node: Node = None
             self.runbook_name: str = None
-            self.image_name: str = None
```

Comment:
the value in this variable is actually the environment name; renaming accordingly